demos/datepicker/localization.html

```diff
@@ -4,23 +4,15 @@
 	<meta charset="utf-8">
 	<title>jQuery UI Datepicker - Localize calendar</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/datepicker.js"></script>
-	<script src="datepicker-ar.js"></script>
-	<script src="datepicker-fr.js"></script>
-	<script src="datepicker-he.js"></script>
-	<script src="datepicker-zh-TW.js"></script>
 	<link rel="stylesheet" href="../demos.css">
-	<script>
-	$(function() {
+	<script src="../require-config.js"></script>
+	<script src="../../external/requirejs/require.js"></script>
+	<script src="../bootstrap.js" data-modules="datepicker-ar.js datepicker-fr.js datepicker-he.js datepicker-zh-TW.js">
```

Comment:
Yes its to load them via url and avoid the base since loading files from within the demos is rare ( maybe only this one ) where as requiring modules from ui is in many many demos